husky_components/datagrid/main.js

```diff
@@ -495,6 +495,24 @@
             },
 
             /**
+             * deselects an item with a given id
+             * @event husky.datagrid.deselect.item
+             * @param {String|Number} The id of the item to deselect
+             */
+            DESELECT_ITEM = function() {
+                return this.createEventName('deselect.item');
```

Comment:
hmm all other events in the datagrid has the same style (for my opinion also wrong) - like `husky.datagrid.select.item`. and i think we should`t change all event names ...